src/utils/http.ts

```diff
@@ -64,15 +69,15 @@ export function makeCacheHeaders(
   }
   headers['cache-control'] = [{ value: cacheControl.join(', ') }]
 
-  // When the browser cache life is longer than the CDN cache life, the "If-Modified" check will always return "true",
-  // so the Last-Modified header affects nothing.
-  if (browserCacheTime < cdnCacheDuration) {
-    headers['last-modified'] = [{ value: new Date().toUTCString() }]
-  }
-
   return headers
 }
 
+export function getBodyHash(body: string): string {
+  // SHA1 is the fastest built-in hash. It takes about 1ms to hash 100KB on a 1-core Lambda@Edge.
+  // When the SHA1 algorithm is used, the last symbol is always `=` that can be removed.
```

Comment:
`base64` uses `=` as padding, it can also be 2 `=` signs or none. If we want to trim those, we should do it explicitly and not slice the last character in all cases.

Reply:
I was realizing it when I wrote this code. I've decided to do it this way here because the length of a SHA-1 hash is always the same, so the last character is always `=`. If you object, I can implement a universal trimming, no problem. What do you think?

Reply:
I wasn't sure if the padding depends on the specific last 3 bytes in the SHA1 output, but it seems it's only about length, so I think you are right. We can keep it like this.

Reply:
The length of the base64 padding depends only on the length of the input